routes/index.js

```diff
@@ -24,5 +24,6 @@ app.use("/levels", require("./levels.js"));
 app.use("/items", require("./items.js"));
 app.use("/cache", require("./cloudflareCache.js"));
 app.use("/external-accounts", require("./external-accounts.js"));
+app.use("/rooms", require("./events.js"));
```

Comment:
`/rooms` as an endpoint and events as controller and modal is confusing

Reply:
Yes, we have discussed about this but haven't reached the conclusion on how should we proceed with it.